django/db/migrations/autodetector.py

```diff
@@ -436,6 +489,42 @@ def generate_renamed_models(self):
                             self.old_model_keys.append((app_label, model_name))
                             break
 
+    def m2m_field_compare(self, through, rem_through):
```

Comment:
Can you move that function above `only_relation_agnostic_fields()` and a brief docstring.